app/assets/javascripts/discourse/app/components/user-nav.js

```diff
@@ -2,12 +2,14 @@ import I18n from "I18n";
 
 import Component from "@glimmer/component";
 import { inject as service } from "@ember/service";
+import { tracked } from "@glimmer/tracking";
 
 export default class UserNav extends Component {
   @service currentUser;
   @service site;
   @service router;
 
+  @tracked
```

Comment:
Hm, tracking a getter? That doesn't sound right! 😃

Reply:
O lol I’ve been dealing with old components and glimmer components and gold confused.